public/js/CoinWrapperBuySignal.js

```diff
@@ -301,6 +301,64 @@ class CoinWrapperBuySignal extends React.Component {
       );
     });
 
+    const buyNextGrid = () => {
+      const sellGridTrade = symbolConfiguration.sell.gridTrade;
+
+      const currentPrice = parseFloat(buy.currentPrice)
+      const lastBuyPrice = parseFloat(sell.lastBuyPrice)
+
+      if (isNaN(lastBuyPrice) || sellGridTrade.length != 1 || currentPrice >= lastBuyPrice)
+        return ('')
+
+      const totalBoughtQty = gridTrade
+          .filter(trade => trade.executed)
+          .map(order => parseFloat(order.executedOrder.executedQty))
+          .reduce((acc, qty) => acc + qty, 0);
+
+      const triggerSellPercentage = sellGridTrade[0].triggerPercentage;
+
+      const gain = triggerSellPercentage - 1;
+
+      const nextGridQty = -totalBoughtQty * (1 + ((currentPrice - lastBuyPrice) / (gain * currentPrice)));
+
+      const nextGridAmount = nextGridQty * currentPrice;
+
+      return (
+        <React.Fragment
+            key={'coin-wrapper-buy-next-grid-row-' + symbol}>
+          <div className='coin-info-column coin-info-column-price'>
+            <span className='coin-info-label'>
+              &#62; Next grid amount:
```

Comment:
So this is a suggested grid amount, not the amount the bot will execute it. 
Can we rename the label to `Suggested next grid amount`?

Reply:
Sure @chrisleekr. I have edited the PR description to clarify this is only a suggestion.

Reply:
@chrisleekr, after more thought, I propose to rename 'Suggested break-even amount', which is really about closing the grid without losing money if the market moves up only the sell.triggerPercentage.